components/CSVTable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
```

Comment:
Change this to import everything on one line like `import { Table, TableBody, TableCell, etc. } from '@mui/material'`